distribution/changelog.txt

```diff
@@ -1,6 +1,6 @@
 0.4.6 (in development)
 ------------------------------------------------------------------------
-- Feature: [Plugin] Added API for returning metadata from all registered plugins.
+- Feature: [#20709] [Plugin] Plugins can now check metadata from all registered plugins.
```

Comment:
We sort the changelog entries by the issue/PR number, please move it to the correct line.

Reply:
Understood, my bad. Moved it to the correct location in the latest commit.